index.php

```diff
@@ -25,8 +29,27 @@
                                 </blockquote>
 
                                 <!-- Chart here -->
+                                <div id="svgContainer">
+                                    <svg xmlns="http://www.w3.org/2000/svg"></svg>
+                                </div>
+                                <style>
+                                    /* Added custom CSS for the svgContainer */
+                                    #svgContainer {
+                                        overflow: auto;
+                                        max-width: 100%; /* Adjust as needed */
+                                        max-height: 60vh; /* Adjust as needed */
+                                    }
 
-                                
+                                    /* Added styles for the inner svg */
+                                    #svgContainer svg {
+                                        min-width: 1200px; /* Set a minimum width to enable horizontal scrolling */
+                                        min-height: 1600px; /* Set a minimum height to enable vertical scrolling */
+                                    }
+                                </style>
```

Comment:
This should probably be in an external CSS file

Reply:
Deferred to #23